packages/coin-support-utils/tests/02.createAccount/02.generateDerivationPathPerScheme/__fixtures__/types.ts

```diff
@@ -0,0 +1,17 @@
+import { IGenerateDerivationPathsPerSchemeParams } from '../../../../src/createAccount';
+
+export interface DerivationPathPerSchemeGeneratorTestCases {
+  name: string;
+  input: IGenerateDerivationPathsPerSchemeParams;
+  output: Record<
+    string,
+    {
+      derivationPath: string;
+      index: number;
+    }[]
+  >;
```

Comment:
Don't redefine types

```suggestion
  output: ReturnType<typeof generateDerivationPathsPerScheme>;
```